app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -7,14 +7,15 @@ def host_overview_buttons(host)
     end
 
     def multiple_actions
-      super + [ [_('Schedule Remote Job'), new_job_invocation_path, false] ]
+      super + [ [_('Schedule Remote Job'), new_job_invocation_path, false] ] +
+              user_job_actions.map { |a| a.push(false) }
     end
 
     def schedule_job_multi_button(*args)
-      host_features = rex_host_features(*args)
+      items = [rex_host_features(*args), user_job_actions.map { |a| link_to(a[0], a[1]) }].flatten.compact
```

Comment:
Could we destructure the array instead of taking it as a whole and then accessing elements by indexes?